keybot/win_jenkins/win_jenkins.go

```diff
@@ -0,0 +1,173 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package win_jenkins
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+)
+
+const jenkinsUrl string = "http://192.168.1.10:8080"
```

Comment:
dont need string... `const jenkisUrl`... and also URL not Url...

you should install pre-commit hook for this repo.. it runs metalinter